packages/data-stores/src/site/actions.ts

```diff
@@ -95,25 +95,36 @@ export function createActions( clientCreds: WpcomClientCredentials ) {
 		type: 'RESET_RECEIVE_NEW_SITE_FAILED' as const,
 	} );
 
-	const launchSiteStart = ( siteId: number ) => ( {
-		type: 'LAUNCH_SITE_START' as const,
+	const launchSiteIdle = ( siteId: number ) => ( {
+		type: 'LAUNCH_SITE_IDLE' as const,
```

Comment:
I believe that, as an action, `launchSiteStart` was not a bad name — in fact, it's called when we're starting the site launch procedure